### PR TITLE
Css flexbox

### PR DESCRIPTION
**Pull request by claylevy53**
Screen Shot 2023-11-27 at 8 45 10 PM
Screen Shot 2023-11-27 at 8 45 42 PM
Screen Shot 2023-11-27 at 8 46 15 PM
Screen Shot 2023-11-27 at 8 47 08 PM

### CODE REVIEW

File: css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+
+      #card-deck {
+        display: flex;
+        min-height: 100vh;
+        /* align-items: center; */
```
Comment (kurtyazdizadeh) [inline css-flexbox/align-items.html]: you should uncomment this :)

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
See comment for changes!

File: css-flexbox/align-items.html (hunk 1 above)
Comment (claylevy53) [reply css-flexbox/align-items.html]: New version merged with fixes!

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Pull request merged**